### PR TITLE
RELATED: ONE-4697 Explicitly mention infinite scrolling with the tabl…

### PR DESCRIPTION
**Pull request by vlastikcz**
…e column auto sizing

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/499/ : SUCCESS in 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/292/ : SUCCESS in 4s

**Review by patrikbraborec — APPROVED** [review]

File: docs/10_vis__pivot_table_component.md [hunk 1]
```diff
@@ -150,12 +150,14 @@ columnSizing: {
...
 * The maximum column width is 500 px.
-* All the columns are automatically resized to fit their content.
 * New columns (that is, those that are shown after the initial render) are resized only if the table has not been vertically or horizontally scrolled. This is useful especially for responsive tables with a dynamic width.
 * A change of attributes, measures, filters, or totals in the table is handled as a new table. After the change is made, the column size is re-calculated based on the new data.
 * Scrolling horizontally or vertically and sorting values in a column do not affect the column width.
 * If you manually adjust the column width, the adjusted width is preserved only temporarily and will be reset to the previously set value after the table is re-rendered.
 
+#### Infinite scrolling
+Tables which use infinite scrolling (dynamic data loading of paginated data) are resized based on the first batch of rows. Dynamically loaded rows do not modify the column sizes after the initial data load.
```
Comment (evgeniabogranskaya) [inline docs/10_vis__pivot_table_component.md:159]: Tables that use infinite scrolling (loading content continuously as a user scrolls down the page) are resized based on the first batch of rows. Dynamically loaded rows do not modify the size of the columns after the initial data load.

**Comment by evgeniabogranskaya** [comment]
Suggested minor edits - LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/500/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/293/ : SUCCESS in 6s

**Review by patrikbraborec — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/501/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/48/ : SUCCESS in 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/140/ : SUCCESS in 16s

**Pull request merged**